packages/yoga/package.json

```diff
@@ -39,7 +39,8 @@
     "@gympass/yoga-tokens": "^0.3.0",
     "@ptomasroos/react-native-multi-slider": "^1.0.0",
     "prop-types": "^15.7.2",
-    "rc-slider": "^8.7.1"
+    "rc-slider": "^8.7.1",
+    "react-google-font-loader": "^1.1.0"
```

Comment:
Since this library does not have any tests, do you think we should add some to verify if the tag `link` is created properly?

Reply:
Yes, great point of view, I've done this